### PR TITLE
Mini Village Map

### PR DESCRIPTION
**Pull request by AnyBlue**
New map made by Elvis26 and DiamondPlane

### CODE REVIEW

**Comment by Avyukt1** [comment]
Good Map!

**Review by Panquesito7 — COMMENTED** [review]
IMHO, this directory name should be `mini_village`.

**Comment by AnyBlue** [comment]
> IMHO, this directory name should be `mini_village`.

@Panquesito7 I don' know how to edit it...

**Comment by AnyBlue** [comment]
Oh, yay! I did it

File: mini_village/map.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+team.1.color = red
+team.1 = red
+team.1.pos = (-8, -8, 101)
+team.2.pos = (0, -8, -101)
+team.2.color = blue
+author = DiamondPlane And Elvis26
```
Comment (JostP) [inline mini_village/map.conf:6]:
```suggestion
author = DiamondPlane and Elvis26
```

Comment (AnyBlue) [reply mini_village/map.conf:6]: Ok

**Comment by JostP** [comment]
I reviewed your map. Map looks interesting :+1: 
Here are some bugs I found:
- There are holes in red barrier. Please remove them.
Zaslonska slika 2020-12-03 19-10-22
- There is grass in towers and it is not supposed for it to be there. Remove it from towers and from rocket.
Zaslonska slika 2020-12-03 19-08-51
- Your screenshot is not really good and it is not correct size (should be 900x600 px = ratio 3:2). Here I created on for you.
Zaslonska slika 2020-12-03 19-00-24

**Review by JostP — APPROVED** [review]
Looks good :+1:

**Pull request merged**